pyproject.toml

```diff
@@ -23,18 +23,21 @@ dependencies = [
     "evaluate",
     "datasets>=2.16.0",
     "evaluate>=0.4.0",
+    "jaxtyping",
     "jsonlines",
     "numexpr",
     "peft>=0.2.0",
     "pybind11>=2.6.2",
     "pytablewriter",
     "rouge-score>=0.0.4",
     "sacrebleu>=1.5.0",
+    "sae_lens",
```

Comment:
Adding a hard dependency on both SAE lens and transformer lens to the eval harness seems unnecessary. We should be able to make this optional.